packages/slice-machine/src/domain/customType.ts

```diff
@@ -90,13 +90,41 @@ export function getMainSectionEntry(
   return sections[0];
 }
 
+export function getMainSectionId(customType: CustomType): string | undefined {
+  return Object.keys(customType.json)[0];
+}
+
 export function getSection(
   customType: CustomType,
   sectionId: string,
 ): DynamicSection | undefined {
   return customType.json[sectionId];
 }
 
+export function getSectionWithUIDField(
```

Comment:
💡 #idea

You can see in the file that there is a pattern to name with entry when it's an array return so it's clear that it's not direction the section value itself

Reply:
true, will rename it